exporter/otelarrowexporter/internal/arrow/grpcmock/credentials.go

```diff
@@ -0,0 +1,74 @@
+// Code generated by MockGen. DO NOT EDIT.
```

Comment:
Not sure about this generated code. Shouldn't this have its `go:generate` clause? When should this be re-generated? How can the CI confirm the generated code hasn't been manually modified?